File: Examples/Example-Mac/PINRemoteImage/APNGViewController.m
//
//  APNGViewController.m
//  PINRemoteImage
//
//  Created by Garrett Moon on 4/23/18.
//  Copyright © 2018 mischneider. All rights reserved.
//

#import "APNGViewController.h"

#import "PINAnimatedImageView+PINRemoteImage.h"

@interface APNGViewController ()

@property (weak) IBOutlet PINAnimatedImageView *animatedImageView;

@end

@implementation APNGViewController

- (void)viewWillAppear {
    [super viewWillAppear];
    [self.animatedImageView pin_setImageFromURL:[NSURL URLWithString:@"https://upload.wikimedia.org/wikipedia/commons/1/14/Animated_PNG_example_bouncing_beach_ball.png"]];
}

@end

File: Examples/Swift-Example/Swift-Example/APNGViewController.swift
//
//  APNGViewController.swift
//  Swift-Example
//
//  Created by Garrett Moon on 4/24/18.
//  Copyright © 2018 Marius Landwehr. All rights reserved.
//

import UIKit
import PINRemoteImage

class APNGViewController: UIViewController {
    
    @IBOutlet weak var imageView: PINAnimatedImageView!
    
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        
        imageView.pin_setImage(from: URL(string: "https://upload.wikimedia.org/wikipedia/commons/1/14/Animated_PNG_example_bouncing_beach_ball.png"))
    }
}
